### ISSUE TITLE
Convert Guitar Component to Accessible Grid

### ISSUE THREAD

**Issue by TomChamberlainUK** [TomChamberlainUK/music#88]
# Convert Guitar Component to Accessible Grid

Set the `<Guitar />` component to use grid elements to maximise accessibility

## Description

Currently our `<Guitar />` component looks visibly suitable but fails to be accessible via screen readers.

We should apply grid html element, to make it both keyboard and screenreader navigable.

## Notes

- See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/grid_role

## Done When

- [x] `<Guitar />` is an accessible grid
- [x] `<String />`s are separate accessible grid rows
- [x] `<Fret />`s are accessible grid cells
- [x] ~~`<Guitar />` is screen reader accessible~~ — This can be handled in an accessibility review later
- [x] `<Guitar />` is keyboard accessible

**Comment by TomChamberlainUK** [comment]
As the fretboard would have no column or row headers, I don't think the guitar should actually be a table

It should however be focusable and navigable — I'll update the done criteria

**Comment by TomChamberlainUK** [comment]
Scratch that, I think it should be [a table with the role of grid](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/grid_role)
